docs/index.html

```diff
@@ -1,21 +1,52 @@
-    <p class="p-t-2-xs"><span class="bold">Efficiency</span> Email design is stereotypically a messy, painful experience. This guide takes the guesswork out of email design to make it an easier process. With the templates and reusable components, you no longer have to reinvent the wheel as you embark on creating an email. Common patterns are already defined, and all you have to do is replace the content. Also, there's no more fighting with inline styles, as the CSS is contained in a central place. And to top it all off, you're able to start with a stable foundation that has already been thoroughly tested in our top <a href="{{ site.baseurl}}/atmosphere/email_client_support.html">email clients</a>. Email clients render emails in inconsistent and surprising ways, but no worries—the messy, dirty debugging process has already been done. All this to say—projects will get done quicker than ever, with easy-to-reuse code and less cursing of antiquated email clients.</p>
-  </div>
-  <div class="col col-100-xs col-100-md col-50-lg">
+<div class="col col-100-xs">
+  <div class="col col-100-xs p-b-2-xs">
     <h3 class="p-b-2-xs">How do I use the guide?</h3>
-    <p class="p-b-2-xs"><span class="bold">Where to start</span> Whether you're a marketer, designer, developer, salesperson, or agency, this guide is a resource for building emails at Treehouse. There are a few routes you can take, depending on the type of email you're sending and your desire to work within a WYSIWYG editor or with code. The chart below is a guide which helps walk you through the process. It'll help you determine if the email you're creating should belong in Intercom, MailChimp, Marketo, or Mandrill. Specific instructions for how to create it (in the ESP or outside the ESP) follow.</p>
+    <p class="p-b-2-xs">Whether you're a marketer, designer, developer, salesperson, or agency, this guide is a resource for building emails at Treehouse. There are a few routes you can take, depending on your goals, needs, and tool requirements.</p>
+    <h3 class="p-b-2-xs">Start by checking out the quick start guides</h3>
+    <a class="m-b-05-xs button button--primary button--large-xs m-r-0-xs" href="{{ site.baseurl}}/getting-started.html" style="margin-top: auto;">Get Started</a>
+    <h3 class="p-t-2-xs">Or take a short cut to find what you're looking for</h3>
+  </div>
+</div>
+<div class="col col-100-xs border-t-light-xs border-b-light-xs p-tb-3-xs block-xs block-md flex-block-lg">
+  <div class="col col-100-xs col-50-md col-25-lg p-lr-2-xs text-center-xs flex-block-xs flex-column-xs">
+    <img src="{{ site.baseurl}}/assets/img/illustrations/principles.png" class="flex-self-baseline-xs w-full-xs p-b-1-xs" alt="Learn Principles">
```

Comment:
What do you things about this `alt` attribute.

Reply:
Looks good